### DISCUSSION TITLE
Randomly oriented grains

### DISCUSSION THREAD

**Discussion by LiZa111111111** [neperfepx/neper#571] [Q&A] [answered]
Hello.
There is no problem to generate randomly oriented grain tessellation. However, I wonder why every time I run the random tessellation, it creates exactly the same orientation? If it is a random, it should give different orientations every time. Is there any possibility to shuffle the grain orientations in such way that I get the new orientations every time?
Thanks.

**Comment by rquey** [comment, marked as answer]
Neper only generates one orientation set for one morphology. That is, if you change the `-id` value, the morphology will change and the orientations will change, but they will be the same on every run (for the same `-id`).

If you would like to keep the same morphology but change the orientations, keep the same `-id` value, but use `-ori "file(...)"` to load custom orientations.

**Reply by LiZa111111111** [reply]
Thank you so much! This worked fine and what I was looking for.
